RedHatInsights/insights-inventory-frontend
# feat(RHINENG-9249): create bifrost toggle

Resolves: https://issues.redhat.com/browse/RHINENG-9249

This creates a new toggle to switch between bootc table view and hybrid inventory tabs. This toggle should only be shown to accounts that has bootc images. 

To test:
1. Run the PR
2. Open the app with an account that has bootc images (ping me for an account with bootc)
3. Observe the new toggle is present and by default hybrid tabs are shown
4. Click on the bootc toggle on the right
5. Observe that you see a mock bootc table
6. Open the app with another account that does not have any bootc image
7. Observe that the toggle is not shown
